akka-samples/akka-sample-cluster-scala/build.sbt

```diff
@@ -16,10 +16,11 @@ val project = Project(
       "com.typesafe.akka" %% "akka-actor" % akkaVersion,
       "com.typesafe.akka" %% "akka-remote" % akkaVersion,
       "com.typesafe.akka" %% "akka-cluster" % akkaVersion,
+      "com.typesafe.akka" %% "akka-cluster-metrics" % akkaVersion,
       "com.typesafe.akka" %% "akka-contrib" % akkaVersion,
       "com.typesafe.akka" %% "akka-multi-node-testkit" % akkaVersion,
       "org.scalatest" %% "scalatest" % "2.2.1" % "test",
-      "org.fusesource" % "sigar" % "1.6.4"),
+      "io.kamon" % "sigar-loader" % "1.6.5-rev001"),
     javaOptions in run ++= Seq(
       "-Djava.library.path=./sigar",
```

Comment:
would it be possible to remove the sigar files from the sample now? (that can also be done in a later step)